packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
+
+### Standard view
```

Comment:
Instead of giving these labels, descriptions are sufficient, e.g. `Toolbar with title`